tensorflow/lite/kernels/internal/BUILD

```diff
@@ -952,6 +952,19 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "common_test",
+    srcs = [
+        "common_test.cc",
+    ],
+    shard_count = 2,
```

Comment:
nit: we can remove this since the total test execution time will be quite small.

Reply:
Updated 😃